lib/logger.js

```diff
@@ -0,0 +1,17 @@
+const debug = require("util").debuglog("riff");
+
+class Logger {
```

Comment:
do we really need to define a new class and force everyone to require it? Why can't we use console? What if someone (or some library) does use console?

Reply:
I think introducing a class makes sense, as we can e.g. introduce common logging vocabulary, regardless of the underlying resource.
If we want one day to change the formatting of error messages, there will be a single place to change vs. modifying `console.error` scattered all over the place.

Reply:
I also can configure https://eslint.org/docs/rules/no-console to make it even more explicit that `Logger` is the class to use.

Reply:
> I think introducing a class makes sense, as we can e.g. introduce common logging vocabulary, regardless of the underlying resource.

Node has a common logging vocabulary, it's called `console`.

> If we want one day to change the formatting of error messages, there will be a single place to change vs. modifying console.error scattered all over the place.

Strong disagree. We can code against the `console` object and manage the behavior at runtime. Using `console` would actually clean up our current code because we could centralize the debug logging behind `console.trace()`.


Early in server.js we could do something like:

```js
global.console = new console.Console(process.stdin, process.stderr);
if (muteTrace) {
  global.console.trace = () => {}
}
```

Reply:
Having a central class is also useful for the integration of debuglog, especially with its yet-to-come second usage: https://github.com/projectriff/node-function-invoker/issues/159.

Clients will just have to call a single method and not have to worry about remembering the key.

Reply:
Using console directly now.